### PR TITLE
Fix description of changing effective roles in scope mappings

### PR DESCRIPTION
**Pull request by little-pinecone**
* replace 'ON' with 'OFF' when specifying the 'Full Scope Allowed' option value that enables changing the default mappings

### CODE REVIEW

File: server_admin/topics/roles-groups/con-role-scope-mappings.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ By default, each client gets all the role mappings of the user. You can view the
 .Full scope
 image:{project_images}/full-client-scope.png[Full scope]
 
-By default, the effective roles of scopes are every declared role in the realm. To change this default behavior, toggle *Full Scope Allowed* to *ON* and declare the specific roles you want in each client.  You can also use <<_client_scopes, client scopes>> to define the same role scope mappings for a set of clients.
+By default, the effective roles of scopes are every declared role in the realm. To change this default behavior, set the *Full Scope Allowed* option to *OFF* and declare the specific roles you want in each client.  You can also use <<_client_scopes, client scopes>> to define the same role scope mappings for a set of clients.
```
Comment (andymunro) [inline server_admin/topics/roles-groups/con-role-scope-mappings.adoc]: @little-pinecone We are standardizing on this format for something you enable: “Toggle *<switch-name>* to *ON*.”

**Review by andymunro — CHANGES_REQUESTED** [review]
One formatting change needed. Thanks.

File: server_admin/topics/roles-groups/con-role-scope-mappings.adoc (hunk 1 above)
Comment (little-pinecone) [reply server_admin/topics/roles-groups/con-role-scope-mappings.adoc]: @andymunro In this particular case we want to disable the default option to be able to declare roles manually. Therefore, I changed it to "_toggle *Full Scope Allowed* to *OFF*_". Is my understanding correct?

**Comment by ahus1** [comment]
@little-pinecone - thank you for this pull request, and sorry it took so long to review this PR. 

This has been fixed in the meantime in commit 858c0a8ae19bb71b7b8c1772bd11aa10d81485eb, therefore I'm closing this PR.

**Pull request closed without merging**